tests/testMask.py

```diff
@@ -349,7 +349,7 @@ def testInterpret(self):
             self.assertEqual(im.interpret(bitmask), p)
             im.getArray()[0, i] = bitmask
             self.assertEqual(im.getAsString(i, 0), p)
-        self.assertEqual(self.Mask.interpret(allBits), ",".join(planes.keys()))
+        self.assertEqual(self.Mask.interpret(allBits), ",".join(sorted(planes.keys())))
```

Comment:
Just for the sake of keeping a record: the issue is that in C++ MaskPlanes are maps, which are essentially dictionaries with sorted keys. SWIG mapped this to a dict with sorted keys while pybind11 maps them to ordinary dicts. @RobertLuptonTheGood confirmed that MaskPlanes do not need to be sorted, so it is ok to just change the API so that it does not expect MaskPlane.keys() to be sorted.